### PR TITLE
SheetMetalUnfolder.py: refinements

### PR DESCRIPTION
**Pull request by luzpaz**
Various cosmetic fixes

### CODE REVIEW

File: SheetMetalUnfolder.py [hunk 1]
```diff
@@ -317,57 +313,57 @@ class Simple_node(object):
...
-    self.node_flattened_faces = [] # faces of a flattened bend node.
-    self.unfoldTopList = None # source of identical side edges
-    self.unfoldCounterList = None # source of identical side edges
-    self.actual_angle = None # state of angle in refolded sheet metal part
-    self.p_wire = None # wire common with parent node, used for bend node
-    self.c_wire = None # wire common with child node, used for bend node
-    self.b_edges = [] # list of edges in a bend node, that needs to be recalculated, at unfolding
+    self.node_flattened_faces = [] # Faces of a flattened bend node
+    self.unfoldTopList = None      # Source of identical side edges
+    self.unfoldCounterList = None  # Source of identical side edges
```
Comment (luzpaz) [inline SheetMetalUnfolder.py:348]: I think the comment here is incorrect

**Comment by shaise** [comment]
Will merge this pull, but will get the same issues the next time I take Ulrich's unfolder script.

**Comment by luzpaz** [comment]
can we upstream the changes to @ulrich1a ?

**Comment by shaise** [comment]
don't really know.

**Pull request merged**